src/term_impls.rs

```diff
@@ -182,6 +185,49 @@ where
     }
 }
 
+macro_rules! impl_iter_term {
+  ($iter:ty) => {
+        impl <T> Term for $iter where
+        $iter: IntoIterator < Item = T> + FromIterator < T >,
+        for < 'b> & 'b $iter: IntoIterator < Item = &'b T >,
+        for < 'b > & 'b mut $iter: IntoIterator < Item = & 'b mut T >,
+        T: Term
+        {
+        fn map_one_transform < F > ( self, f: & mut F) -> $iter where
+        F: GenericTransform {
+        self.into_iter().map( | x| f.transform(x)).collect()
```

Comment:
Oof... Is this what `rustfmt` does to the macro's formatting? Are you using `rustfmt` or `rustfmt-nightly`?